pulsar-broker-common/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGeneratorUtils.java

```diff
@@ -88,19 +88,27 @@ public static void generateSystemMetrics(SimpleTextOutputStream stream, String c
         }
     }
 
+    static String getTypeNameSuffix(Collector.Type type) {
+        if (type.equals(Collector.Type.INFO)) {
+            return "_info";
+        }
+        return "";
```

Comment:
This might be a premature optimization, but the `stream.write` method accepts null values, so it might be better not to create the empty string here.

```suggestion
        return null;
```